pycode/test/test_epidata_getCommuterMobility.py

```diff
@@ -222,90 +263,140 @@ def write_commuter_all_federal_states(self, out_folder):
-        counties = gD.loadExcel(
-            'kreise_deu', apiUrl=self.path, extension='.xlsx',
-            param_dict={"sheet_name": 1})
-        setup_dict = {'num_counties': 2,
-                      'abs_tol': 100,
-                      'rel_tol': 0.01,
-                      'num_govregions': 2,
-                      'counties': counties,
-                      'path': self.path}
-        cm.map_keys_to_numlists(setup_dict)
-        mock_print.assert_any_call('Error. Number of government regions wrong. Having', 17, 'instead of', 2)
-        mock_print.assert_any_call("Error. Number of counties wrong.")
-        mock_print.assert_any_call('Error. Number of governing regions wrong.')
-
-        mock_print.call_args_list = []
-        cm.assign_geographical_entities(setup_dict)
-        mock_print.assert_any_call('Error. Number of government regions wrong.')
-
-        mock_print.call_args_list = []
-        cm.verify_sorted(countykey_list=pd.Series(['01001', '01053', '02000', '03101', '06632', '04012', '05112',
-                                                   '05316', '06532', '07141']))
-        mock_print.assert_any_call('Error. Input list not sorted, population per county list had to be sorted '
-                                   'accordingly.')
-
-        mock_print.call_args_list = []
-        setup_dict['num_counties'] = 21
-        setup_dict['num_govregions'] = 17
-        cm.get_matrix_commuter_migration_patterns(setup_dict)
-        mock_print.assert_any_call('Error in calculations for county ', 'Duisburg, Stadt', '\nAccumulated values:',
-                                   333.0, ', correct sum:', 305.0)
-
-    def test_migration_data(self):
+    
+    def test_commuter_data(self):
         """! Tests migration data by some randomly chosen tests.
         """
+        
+        countykey2numlist = collections.OrderedDict(
+            zip(self.countykey_list, list(range(0, len(self.countykey_list)))))
+
         gD.check_dir(self.path)
         self.write_kreise_deu_data(self.path)
         self.write_commuter_all_federal_states(self.path)
         self.assertEqual(len(os.listdir(self.path)), 17)
-        counties = gD.loadExcel(
-            'kreise_deu', apiUrl=self.path, extension='.xlsx',
-            param_dict={"sheet_name": 1})
-        setup_dict = {'num_counties': 21,
-                      'abs_tol': 100,
-                      'rel_tol': 0.01,
-                      'num_govregions': 17,
-                      'counties': counties,
-                      'path': self.path}
-        (countykey_list, countypop_list, govkey_list, countykey2numlist, govkey2numlist, gov_county_table,
-         countykey2govkey,
-         countykey2localnumlist, state_gov_table, mat_commuter_migration) = cm.get_data(setup_dict)
+        
         # just do some tests on randomly chosen migrations
 
-        # check migration from Leverkusen (averaged from NRW, 05) to Hildburghausen
```

Comment:
Please add this functionality testing again. The averaging from a state should be tested.